Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -390,7 +390,7 @@ const ScrollView = React.createClass({
 
   _handleScroll: function(e: Object) {
     if (__DEV__) {
-      if (this.props.onScroll && !this.props.scrollEventThrottle && Platform.OS === 'ios') {
+      if (this.props.onScroll && this.props.scrollEventThrottle == null && Platform.OS === 'ios') {
```

Comment:
@janicduplessis @dustin-H isn't it going to be `undefined` instead of `null` when somebody forgets to pass in `scrollEventThrottle`

wondering if there are any downsides to just defauting this to 16 and removing this warning.